### PR TITLE
updates clojurebridge stuff-page

### PR DESCRIPTION
**Pull request by yokolet**
This pull request updates the page, https://github.com/ClojureBridge/board/blob/master/ClojureBridge%20Stuff%20%26%20Where%20it%20Lives.md . Please review and merge this.

### CODE REVIEW

File: ClojureBridge Stuff & Where it Lives.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ All accounts are free unless otherwise noted.
 
 | Account | Who Has the Credentials | Notes |
 |---------|-------------------------|-------|
-| Twitter | Yoko                    | @ClojureBridge |
-| Mailchimp | Bridget, Jen, Jamie    | Email response(?) when people subscribe on clojurebridge.org |
+| Twitter | Board members           | @ClojureBridge |
+| Mailchimp[[*]](#mailchimp) | <strike>Bridget, Jen, Jamie    | Email response(?) when people subscribe on clojurebridge.org</strike> |
```
Comment (davidchambers) [inline ClojureBridge Stuff & Where it Lives.md]: It's not valid to have a `<strike>` span multiple `<td>` elements. Each `<td>` needs its own `<strike>`.

File: ClojureBridge Stuff & Where it Lives.md [hunk 2]
```diff
@@ -6,14 +6,14 @@ All accounts are free unless otherwise noted.
...
 | G Suite[[*1]](#googleapps) | Each memember  | All board members have name@clojurebridge.org email. Passwords are their own password. |
 | Github | Organization account     | All borad member have an owner status. |
-| Web site | Jamie, Yoko | clojurebridge.org. Adding a new events on Rails console and deploying app to Heroku. Cognitect has the Heroku account. |
-| Domain | Yoko | Hosted at Google. Yoko covered the first year's cost of $12. |
+| Web site | Board members | clojurebridge.org. To add a new events push the page to the clojurebridge.github.io repo. |
+| Domain | Bridge Foundry | Hosted at DNSimple. Free since Bridge Foundry has a free to use NPO account. |
 | Google Groups | Board members | All board member have an owner status. ClojureBridge has board, Workshops, curriculum, and Alumni mailing lists.|
-| Cognitect Resources | Russ Olsen | Github @russolsen , clojurebridge.org is hosted at Cognitect Heorku account. Sponsor logs are uploaded to Congnitect owned S3 bucket.|
+| Cognitect Resources[[**]](#cognitect) | Russ Olsen | Github @russolsen , clojurebridge.org is hosted at Cognitect Heorku account. Sponsor logs are uploaded to Congnitect owned S3 bucket.|
```
Comment (davidchambers) [inline ClojureBridge Stuff & Where it Lives.md]: Could we use `[Cognitect Resources](#cognitect)` here instead? The `[**]` looks ugly to me.

File: ClojureBridge Stuff & Where it Lives.md (hunk 1 above)
Comment (yokolet) [reply ClojureBridge Stuff & Where it Lives.md]: Yeah, indeed :) I didn't check that. It is fixed now.

File: ClojureBridge Stuff & Where it Lives.md (hunk 2 above)
Comment (yokolet) [reply ClojureBridge Stuff & Where it Lives.md]: Fixed this. Also, Mailchimp reference, [*], was fixed in the same way. But, G Suite reference left as is. This ref is permanent, but other two should be deleted at some point.

**Review by davidchambers — APPROVED** [review]

**Pull request merged**